r/R/arrow-package.R

```diff
@@ -38,21 +38,24 @@ supported_dplyr_methods <- list(
   select = NULL,
   filter = NULL,
   collect = NULL,
-  summarise = NULL,
+  summarise = c(
+    "window functions not currently supported;",
+    'arguments `.drop = FALSE` and `.groups = "rowwise" not supported'
+  ),
   group_by = NULL,
   groups = NULL,
   group_vars = NULL,
   group_by_drop_default = NULL,
   ungroup = NULL,
-  mutate = NULL,
+  mutate = "window functions not currently supported",
```

Comment:
For users, it may be easier to explain that "mutate cannot be used after being grouped".

Reply:
Good point, I'll find other words. `group_by() |> mutate()` does work in some cases, just not if there are aggregations involved (like `x - mean(x)`).

Reply:
Added a few more words, let me know what you think

Reply:
I think it is easier to understand. Thanks.